webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+
+const config = {
+  entry: path.join(__dirname, 'spec/performance-test.jsx'),
```

Comment:
Is it maybe confusing that this webpack config is only used for the performance test? I wonder if we should adjust the name of this file?